code/game/mecha/combat/tank.dm

```diff
@@ -0,0 +1,135 @@
+#define NOMINAL    0
+#define FIRSTRUN   1
+#define POWER      2
+#define DAMAGE     3
+#define IMAGE      4
+#define WEAPONDOWN 5
+
+/obj/mecha/combat/tank
+	name = "light adhomian tank"
+	desc = "The Ha'rron MK.IV light tank is an armored vehicle commonly used by tajaran military forces."
+	icon = 'icons/mecha/mecha_large.dmi'
+	icon_state = "tank"
+	initial_icon = "tank"
+	pixel_x = -16
+	step_in = 5
+	dir_in = 1
+	health = 800
+	deflect_chance = 20
+	damage_absorption = list("brute"=0.4,"fire"=1.1,"bullet"=0.5,"laser"=0.8,"energy"=0.8,"bomb"=0.7)
+	max_temperature = 30000
+	force = 40
+	w_class = 35
+	infra_luminosity = 4
+	wreckage = /obj/effect/decal/mecha_wreckage/tank
+	stepsound = 'sound/mecha/tanktread.ogg'
+	turnsound = 'sound/mecha/tanktread.ogg'
+
+/obj/mecha/combat/tank/Initialize()
+	.= ..()
+	var/obj/item/mecha_parts/mecha_equipment/ME = new /obj/item/mecha_parts/mecha_equipment/weapon/ballistic/lmg
+	ME.attach(src)
+	ME = new /obj/item/mecha_parts/mecha_equipment/weapon/ballistic/cannon
+	ME.attach(src)
+	ME = new /obj/item/mecha_parts/mecha_equipment/tool/passenger
+	ME.attach(src)
+	ME = new /obj/item/mecha_parts/mecha_equipment/armor_booster/antiproj_armor_booster
+	ME.attach(src)
+	return
```

Comment:
Does returning nothing here do bad things?

Reply:
Line 29 is

`.= ..()`
This sets the return value to the return value of ..()